conf/config.properties.example

```diff
@@ -725,7 +725,7 @@ client.cmpdreg.serverSettings.maxSearchTime=180000
 client.cmpdreg.serverSettings.maxSearchResults=5000
 client.cmpdreg.serverSettings.unitTestDB=false
 client.cmpdreg.serverSettings.initalDBLoad=false
-client.cmpdreg.serverSettings.projectRestrictions=false
+client.cmpdreg.serverSettings.projectRestrictions=true
```

Comment:
We should have these enabled by default. I checked every prod server and they are all set to true.